src/landscape/landscapeService.js

```diff
@@ -65,30 +65,32 @@ export const fetchLandscapeToView = (slug, currentUser) => {
     ${landscapeFields}
     ${defaultGroup}
   `;
-  return (
-    terrasoApi
-      .requestGraphQL(query, { slug, accountEmail: currentUser.email })
-      .then(_.get('landscapes.edges[0].node'))
-      .then(landscape => landscape || Promise.reject('not_found'))
-      .then(landscape => ({
-        ..._.omit('defaultGroup', landscape),
-        defaultGroup: getDefaultGroup(landscape),
-      }))
-      // TODO temporarily getting position from openstreetmap API.
-      // This should change when we store landscape polygon.
-      .then(landscape =>
-        gisService.getPlaceInfoByName(landscape.location).then(placeInfo => ({
+  return terrasoApi
+    .requestGraphQL(query, { slug, accountEmail: currentUser.email })
+    .then(_.get('landscapes.edges[0].node'))
+    .then(landscape => landscape || Promise.reject('not_found'))
+    .then(landscape => ({
+      ..._.omit('defaultGroup', landscape),
+      defaultGroup: getDefaultGroup(landscape),
+    }))
+    .then(landscape => ({
+      ...landscape,
+      areaPolygon: landscape.areaPolygon
+        ? JSON.parse(landscape.areaPolygon)
+        : null,
+    }))
+    .then(landscape => {
+      if (landscape.areaPolygon) {
+        return landscape;
+      }
+      // Get Bounding box if no areaPolygon data
```

Comment:
suggestion: expand this comment to provide more context. In what circumstances is areaPolygon data not present?